src/main/java/org/mule/service/http/impl/service/client/GrizzlyHttpClient.java

```diff
@@ -90,6 +93,13 @@
   private static final int DEFAULT_SEND_AND_DEFER_BUFFER_SIZE = KB.toBytes(10);
   private static final String DEFAULT_DECOMPRESS_PROPERTY_NAME = SYSTEM_PROPERTY_PREFIX + "http.client.decompress";
 
+  private static final String DISABLE_REQUEST_STREAMING_PROPERTY_NAME = SYSTEM_PROPERTY_PREFIX + "http.requestStreaming.disable";
+  private static boolean requestStreamingDisabled = getProperties().contains(DISABLE_REQUEST_STREAMING_PROPERTY_NAME);
```

Comment:
i found this: https://github.com/mulesoft/mule/blob/mule-4.x/modules/deployment/src/main/java/org/mule/runtime/module/deployment/internal/MuleDeploymentService.java#L125

However, it is using `containsKey` instead of `contains`